lib/chewy/type/import.rb

```diff
@@ -67,6 +67,7 @@ module ClassMethods
         # @option options [String] suffix an index name suffix, used for zero-downtime reset mostly, no suffix by default
         # @option options [Integer] bulk_size bulk API chunk size in bytes; if passed, the request is performed several times for each chunk, empty by default
         # @option options [Integer] batch_size passed to the adapter import method, used to split imported objects in chunks, 1000 by default
+        # @option options [Boolean] direct_import passed to the adapter import method, used skip the collection reload
```

Comment:
The wording in https://github.com/sideqik/chewy/commit/21df05b77eb8ec72cf09811ea4dcd8919080c4f6 seems more clear to me